signalfx/client.py

```diff
@@ -23,13 +24,19 @@ class BaseSignalFx(object):
 
     def __init__(self, api_token, ingest_endpoint=DEFAULT_INGEST_ENDPOINT,
                  api_endpoint=DEFAULT_API_ENDPOINT, timeout=DEFAULT_TIMEOUT,
-                 batch_size=DEFAULT_BATCH_SIZE, user_agents=[]):
+                 batch_size=DEFAULT_BATCH_SIZE, user_agents=None,
+                 enable_aws_unique_id=False):
```

Comment:
Couple reasons- it makes it consistent with the java and ruby libraries which use a parameter to the constructor. It also leaves the choice up to the end user if they want to enable it or not (while defaulting to False) rather than it being hard-coded in the constants file.